### ISSUE TITLE
[Feature] Make this extension available for VSCode web

### ISSUE THREAD

**Issue by shivapoudel** [johnbillion/vscode-wordpress-hooks#31]
I'm often using VSCode web on GitHub and would love to have the same feature available.

**Comment by johnbillion** [comment]
Thanks for the request. Can you provide some info about VS Code web please? Do you mean Codespaces?

**Comment by shivapoudel** [comment]
@johnbillion Yeah, I mean the Codespaces. Maybe this reference will be helpful:
https://github.com/PKief/vscode-material-product-icons/issues/20#issuecomment-900107650
https://github.com/microsoft/vscode/issues/130628

**Comment by shivapoudel** [comment]
@johnbillion Here goes some PR from PHPBlocker extension adopting this feature and thought this might help here. Here goes the tickets and PRs:
- Issue - https://github.com/neild3r/vscode-php-docblocker/issues/176
- PRs - https://github.com/neild3r/vscode-php-docblocker/pull/179 and https://github.com/neild3r/vscode-php-docblocker/pull/179

**Comment by johnbillion** [comment]
Thanks for the info, that's very helpful.
